S4TF_Tutorial_1.ipynb

```diff
@@ -541,11 +541,11 @@
         "* Many of the exercises in the course executes more quickly by using GPU runtime: Runtime | Change runtime type | Hardware accelerator | GPU\n",
```

Comment:
Above this, I'd change

"All usage of Colab in this course is completely free or charge. Even GPU usage is provided free of charge for some hours of usage every day."

to 

"All usage of Colab in this course is completely free of charge. Even GPU usage in Colab is provided free of charge for up to several hours of execution every day."